examples/PCovR_Scaling.ipynb

```diff
@@ -0,0 +1,286 @@
+      "text/plain": [
+       "<Figure size 576x720 with 4 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pcovr_unscaled = PCovR(mixing=0.5).fit(X, y)\n",
+    "T_unscaled = pcovr_unscaled.transform(X)\n",
+    "Yp_unscaled = pcovr_unscaled.predict(X)\n",
+    "\n",
+    "pcovr_scaled = PCovR(mixing=0.5).fit(X_scaled, y_scaled)\n",
+    "T_scaled = pcovr_scaled.transform(X_scaled)\n",
+    "Yp_scaled = y_scaler.inverse_transform(pcovr_scaled.predict(X_scaled))\n",
+    "\n",
+    "\n",
+    "fig, ((ax1_T, ax2_T), (ax1_Y, ax2_Y)) = plt.subplots(2, 2, figsize=(8, 10))\n",
+    "\n",
+    "ax1_T.scatter(T_unscaled[:, 0], T_unscaled[:, 1], c=y, cmap=\"plasma\", ec=\"k\")\n",
+    "ax1_T.set_xlabel(\"PCov1\")\n",
+    "ax1_T.set_ylabel(\"PCov2\")\n",
+    "ax1_T.set_title(\"Latent Projection\\nWithout Scaling\")\n",
+    "\n",
+    "ax2_T.scatter(T_scaled[:, 0], T_scaled[:, 1], c=y, cmap=\"plasma\", ec=\"k\")\n",
+    "ax2_T.set_xlabel(\"PCov1\")\n",
+    "ax2_T.set_ylabel(\"PCov2\")\n",
+    "ax2_T.set_title(\"Latent Projection\\nWith Scaling\")\n",
+    "\n",
+    "ax1_Y.scatter(Yp_unscaled, y, c=np.abs(y - Yp_unscaled), cmap=\"bone_r\", ec=\"k\")\n",
+    "ax1_Y.plot(ax1_Y.get_xlim(), ax1_Y.get_xlim(), \"r--\")\n",
+    "ax1_Y.set_xlabel(\"True Y, unscaled\")\n",
+    "ax1_Y.set_ylabel(\"Predicted Y, unscaled\")\n",
+    "ax1_Y.set_title(\"Regression\\nWithout Scaling\")\n",
+    "\n",
+    "ax2_Y.scatter(Yp_scaled, y, c=np.abs(y - Yp_scaled), cmap=\"bone_r\", ec=\"k\")\n",
+    "ax2_Y.plot(ax2_Y.get_xlim(), ax2_Y.get_xlim(), \"r--\")\n",
+    "ax2_Y.set_xlabel(\"True Y, unscaled\")\n",
+    "ax2_Y.set_ylabel(\"Predicted Y, unscaled\")\n",
+    "ax2_Y.set_title(\"Regression\\nWith Scaling\")\n",
+    "\n",
+    "fig.subplots_adjust(hspace=0.5, wspace=0.3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Also, we see that the total loss (loss in recreating the original dataset and regression loss) does not vary with `mixing` as expected when the datasets are unscaled."
```

Comment:
How about this?
```suggestion
    "Also, we see that the total loss (loss in recreating the original dataset and regression loss) does not vary with `mixing` as expected when the datasets are unscaled. Typically, the regression loss should _gradually_ increase with `mixing` (and vice-versa for the loss in reconstructing the original features). When the inputs are not scaled, however, there will often be a sharp drop or jump in the losses as `mixing` goes to 0 or 1, depending on which input is dominating the results of the model. Here, because the features dominate the model, this jump occurs as `mixing` goes to 0."
```

Reply:
👍